.github/workflows/merge_stage_test.yml

```diff
@@ -0,0 +1,179 @@
+name: merge_stage_test
```

Comment:
Why need this script?

Reply:
the CI behavior is different in pr stage and merge state, we use this script to run different test